src/oscar/templates/oscar/500.html

```diff
@@ -1,15 +1,34 @@
-{% extends "error.html" %}
 {% load i18n %}
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="{{ LANGUAGE_CODE|default:"en-gb" }}" class="{% block html_class %}no-js{% endblock %}">
+    <head>
+        <title>{% block title %}{{ shop_name }} - {{ shop_tagline }}{% endblock %}</title>
```

Comment:
I think this should be hard-coded to "Server error" as it's possible that context processors will not have run depending on what caused the 500 error. 

My view is that we should not use any template tags or template context in this template - it should be pure HTML. See https://github.com/wagtail/wagtail/blob/master/wagtail/project_template/project_name/templates/500.html for example.

Projects can always override this to provide their own styling if they want to.

Reply:
You mean we'd not use even styling at all?

Reply:
Yeah. The rationale for this is to minimise the chance of causing additional errors in the template itself. Django [tries to do the same thing](https://docs.djangoproject.com/en/2.1/ref/views/#django.views.defaults.server_error):

> The default 500 view passes no variables to the `500.html` template and is rendered with an empty `Context` to lessen the chance of additional errors.

In our projects we provide inline CSS for this view, but don't rely on any static assets.

Reply:
Ok agreed.